ci-operator/config/openshift/openshift-tests-private/AGENTS.md

```diff
@@ -0,0 +1,220 @@
+  - Supported platforms: AWS, Azure, Baremetal, GCP, IBMCloud, Nutanix, vSphere
+- Frequency: jobs run periodically, we use `-f{number}` in the job name to define the period
+  - `f28` = every 28 days
+  - `f60` = every 60 days
+- Disconnected: if a job runs on a disconnected environment, its job name contains `disc`
+  - Example: `aws-ipi-disc-priv-sts-f28`
+
+### Filename Examples
+
+**Y Stream Upgrade (4.20 → 4.21):**
+`openshift-openshift-tests-private-release-4.21__amd64-nightly-4.21-upgrade-from-stable-4.20.yaml`
+
+**Z Stream Upgrade (4.20.1 → 4.20.2):**
+`openshift-openshift-tests-private-release-4.20__multi-nightly-4.20-upgrade-from-stable-4.20.yaml`
+
+**Chain Upgrade (4.19 → 4.21):**
+`openshift-openshift-tests-private-release-4.21__arm64-nightly-4.21-upgrade-from-stable-4.19.yaml`
+
+**CPOU Upgrade:**
+`openshift-openshift-tests-private-release-4.20__amd64-nightly-4.20-cpou-upgrade-from-4.18.yaml`
+
+**Rollback:**
+`openshift-openshift-tests-private-release-4.20__amd64-rollback-nightly.yaml`
+
+## Upgrade Job Maintenance Strategies
+
+When maintaining upgrade test jobs, follow these strategic guidelines to ensure comprehensive coverage while avoiding redundancy:
+
+### Common Requirements for all Jobs
+1. **Disconnected profile distribution**
+   - Run most disconnected profiles on **non-multi** architecture
+   - Don't use disconnected profiles for chain upgrade jobs
+
+2. **Multi-architecture preference**
+   - Use multi-architecture for **at least 65%** of profiles
+
+3. **ARM64 preference**
+    - Try to use ARM64 architecture when possible
+
+4. **TechPreview (TP) features**
+    - TP jobs are **only supported for z-stream upgrades**
+
+5. **Architecture-sensitive Features**
+    - For those profiles which include features that are affected by architecture, we copy them on all architectures
+    - The architecture sensitive features include storage (aws efs, gcp filestore-csi, bm LSO), mco, and cluster infrastructure
+
+### Requirements for Chain Upgrade Jobs
+
+1. **Applicable Versions**
+   - Only latest version has Chain upgrade jobs
+
+2. **Cross-platform, cross-feature, cross-architecture coverage**
+   - Chain upgrade paths must cover all platforms: AWS, Azure, Baremetal, GCP, IBMCloud, Nutanix, vSphere
+   - Chain upgrade jobs must cover all key features: sdn (4.15 and earlier; 4.14 default is OVN), sno, ovn, ipsec, ipv4, ipv6, dual-stack, fips, proxy, capability, sts, mixarch
+   - Chain upgrade jobs must cover all CPU architectures: amd64, arm64, multiarch
+
+3. **EOL (End of Life) version handling**
+   - Only keep one more release for EOL versions
+   - Reference: [OpenShift Life Cycle Dates](https://access.redhat.com/support/policy/updates/openshift)
+
+4. **Stable-to-stable upgrade paths**
+   - Choose only ONE profile for stable-to-stable upgrade paths in chain upgrades
+   - The profile should copy from stable-to-nightly
+
+5. **Customer usage-driven coverage**
```

Comment:
suggest to remove the following 3 internal links from this public repo

Reply:
Sure